docs/toubleshooting/DNSBasedGlobalLoadBalancing.md

```diff
@@ -0,0 +1,38 @@
+# DNS Based Global Load Balancing Troubleshooting Guide
+
+This guide provides troubleshooting steps for common issues related to DNS Based Global Load Balancing.
+
+
+## Troubleshoot why TrafficManagerProfile is not programmed
+
+Common reasons and solutions for `TrafficManagerProfile` not being programmed:
+
+1. Invalid resource group or not enough permissions to create/update Azure traffic manager profile in the resource group.
+   - Ensure that the resource group exists and fleet networking controller has been configured correctly to access the resource group.
+2. DNS name is not available.
+   - The DNS may be occupied by other resources. Try to use a different profile name or namespace name.
+3. [Reach the Azure Traffic Manager limits](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#azure-traffic-manager-limits).
+   - 200 profiles are allowed per subscription. If the limit is reached, consider deleting unused profiles or requesting an increase in the limit.

```

Comment:
same comment, how do users know?

Reply:
it should be clearly indicated in the arm response itself and will be included in the status message.